src/sql/workbench/contrib/dashboard/browser/core/dashboardTab.contribution.ts

```diff
@@ -184,56 +180,26 @@ ExtensionsRegistry.registerExtensionPoint<IDashboardTabContrib | IDashboardTabCo
 	}
 });
 
-const tabGroupSchema: IJSONSchema = {
-	type: 'object',
-	properties: {
-		id: {
-			type: 'string',
-			description: localize('azdata.extension.contributes.dashboard.tabGroup.id', "Unique identifier for this tab group.")
-		},
-		title: {
-			type: 'string',
-			description: localize('azdata.extension.contributes.dashboard.tabGroup.title', "Title of the tab group.")
-		}
-	}
-};
-
-const tabGroupContributionSchema: IJSONSchema = {
-	description: localize('azdata.extension.contributes.tabGroups', "Contributes a single or multiple tab groups for users to add to their dashboard."),
-	oneOf: [
-		tabGroupSchema,
-		{
-			type: 'array',
-			items: tabGroupSchema
-		}
-	]
-};
-
-ExtensionsRegistry.registerExtensionPoint<IDashboardTabContrib | IDashboardTabContrib[]>({ extensionPoint: 'dashboard.tabGroups', jsonSchema: tabGroupContributionSchema }).setHandler(extensions => {
-
-	function handleTabGroup(tabgroup: IDashboardTabGroupContrib, extension: IExtensionPointUser<any>) {
-		let { id, title } = tabgroup;
-
-		if (!id) {
-			extension.collector.error(localize('dashboardTabGroup.contribution.noIdError', "No id specified for tab group."));
-			return;
-		}
-
-		if (!title) {
-			extension.collector.error(localize('dashboardTabGroup.contribution.noTitleError', "No title specified for tab group."));
-			return;
-		}
-		registerTabGroup({ id, title });
+const TabGroups: IDashboardTabGroup[] = [
+	{
+		id: 'administration',
```

Comment:
What about custom dashboards that want to have different groups? BDC for example has its services listed under `Cluster Details`.

![image](https://user-images.githubusercontent.com/28519865/78913119-5e2fb500-7a3d-11ea-8230-2a3531b75ba4.png)

It makes sense to have some predefined ones but I still think we should allow custom ones too.

Reply:
this is only for the server/database dashboards, the example you gave is custom modelview dashboard.

Reply:
I think it makes sense to add the tab group contribution back, so that if an extension needs to add a special group, the core doesn't need to know it. meanwhile we provide a good set of predefined groups that will work for most of the scenarios.

Reply:
also updated the group of bdc tab as per @hannahqin and @yualan

Reply:
Yeah I still think that'd be useful. It might get overloaded if a bunch of extensions start making their own groups but I think that's a problem we can concern ourselves with if it ever actually becomes an issue